db/migrate/20150714012236_add_change_field.rb

```diff
@@ -0,0 +1,5 @@
+class AddChangeField < ActiveRecord::Migration
+  def change
+    add_column :runs, :changed_flag, :integer
```

Comment:
I'm thinking this is really "customize_flag"..   Like, what exactly does "changed" mean?